repo2docker/buildpacks/conda/__init__.py

```diff
@@ -154,7 +154,7 @@ def get_build_script_files(self):
             "conda/activate-conda.sh": "/etc/profile.d/activate-conda.sh",
         }
         py_version = self.python_version
-        self.log.info("Building conda environment for python=%s" % py_version)
+        self.log.info("Building conda environment for python={py_version}\n")
```

Comment:
```suggestion
        self.log.info(f"Building conda environment for python={py_version}\n")
```

Reply:
Good catch! This is probably the feature I want most from linters after 'does it compile' syntax checking. F541 (f-string without any placeholders) doesn't improve anything, but its inverse is super useful _and_ not checked-for.

Reply:
It'd be useful, but also tricky to avoid incorrect warnings where `{...}` isn't for formatting.